db/payment_icons.yml

```diff
@@ -941,4 +941,3 @@
 -
   name: bri_direct_debit
   label: BRI Direct Debit
-  group: other
```

Comment:
Seems we actually deleted this line instead of removing the change.

To be clear, we only want the `app/assets/images/payment_icons/mb.svg` changes. `db/payment_icons.yml` is fine as is, doesn't need changing.